packages/terra-consumer-nav/src/components/nav-items/NavItems.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import NavToggler from '../nav-toggler/NavToggler';
+import styles from './NavItems.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  navItems: PropTypes.arrayOf(
+    PropTypes.shape({
+      slug: PropTypes.string,
+      nav_type: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+      target: PropTypes.string,
+      uri: PropTypes.string.isRequired,
+      text: PropTypes.string.isRequired,
+      isActive: PropTypes.bool.isRequired,
+      /*
+      TODO: Need to figure out what we are doing with icons
+
+      icon: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.element,
+      ]),
+      */
+      sub_navs: PropTypes.arrayOf(
```

Comment:
Same comment as above. SubNav should be  a property of a nav item and probalby be a nav item themselves.